### ISSUE TITLE
relations or differences between this project and fakeapp

### ISSUE THREAD

**Issue by xiangcong** [deepfakes/faceswap#347]
Hi~ 
If the original model (the network structure) is the same as what is used in FakeApp(https://www.fakeapp.org)?
Or what's the difference? thanks~!

**Comment by Kirin-kun** [comment]
The models are different. You can't use model files from Fakeapp with Faceswap and vice-versa. As it is closed source, there's no way to tell for sure, but I think that it's a fork of the older original model published in Reddit. I seem to recall it uses the face alignment lib of Adrian Bulat to detect face landmarks, with maybe some tuning. 

I've read here and there that FakeApp was essentially a java gui and packaging of this project, but it's a bit unfair because obviously it has its own features in term of face detection and model.

As for the Fakeapp software, it's not been updated for months, so I don't know if it's dead or if the author is taking his time before releasing something new. 

But at the moment, it's clearly lagging behind this repo.

**Comment by xiangcong** [comment]
Got it~ Thanks for your reply!

**Comment by xiangcong** [comment]
Sorry to bother again.
I just tried tried model of https://anonfile.com/p7w3m0d5be/face-swap.zip
The result is not so good as the following result.
https://www.youtube.com/watch?v=BU9YAHigNx8

Not sure what's the reason? model is too simple?(I'm trying IAE)  small trying dataset?

**Comment by Kirin-kun** [comment]
IAE is particular. There's been a lot of discussion about it. It produces "interesting" faces.

See #383

**Comment by iperov** [comment]
xiangcong because last video is dfaker model

**Comment by xiangcong** [comment]
@iperov  Yeah, I'm just curious about the model dfaker used. Wanner to know the difference between the dfaker model and the models in this project. Because it seems that the last video is quite good~
